src/pages/DataLibrary/components/PACSLookup/QueryBuilder.tsx

```diff
@@ -302,7 +302,7 @@ export const QueryBuilder: React.FC<QueryBuilderProps> = ({
                   <DatePicker
                     placeholder="Study Date (yyyy-MM-dd)"
                     dateFormat={(date: Date) => date.toDateString()}
-                    onChange={(_: any, date: Date) =>
+                    onChange={(_: string, date: Date | undefined) =>
```

Comment:
Same delta as from #694: https://github.com/FNNDSC/ChRIS_ui/pull/694/files#diff-759a989a9f1f37a2e8b45ceedc8f763115a7c24c125e5a3c2d36410d9e8183f4L305-R305

Please leave out these kinds of deltas as they increase the risk of merge conflicts.